src/callbacks/ltfb.cpp

```diff
@@ -263,19 +264,20 @@ void exchange_models__checkpoint_file(lbann_comm& comm,
 void restore_local_model__checkpoint_file(lbann_comm& comm, model& m) {
```

Comment:
Two underscores?

Reply:
Yea, logically I'm trying to convey `{restore_local_model}_{checkpoint_file}`, as opposed to `{restore_local_model}_{some_other_algorithm}`.

Reply:
You should not use two underscores. Technically speaking, any name that contains a double underscore is reserved.

Reply:
It's section 5.10.3.1 of the current draft of the standard if anyone cares. (It's not new, that's just the version I happen to have to look at...)

Reply:
Created issue #1220